docs/using-radicle/pushing-and-pulling-changes.md

```diff
@@ -23,44 +23,44 @@ It can be pushed to Radicle with:
 
 ### Following projects
 
-To contribute to another user's project to your machine, you must first find it on the Radicle network. To find a project, you need it's [**projectID**](understanding-radicle/glossary.md/#project-id). This can be found under in the project page under the [project name](understanding-radicle/glossary.md/#project-name), or in a seed node UI. 
+To contribute to another user's project, you must first find it on the Radicle network. To find a project, you need its [**projectID**](understanding-radicle/glossary.md/#project-id). This can be found in the project page under the [project name](understanding-radicle/glossary.md/#project-name), or in a seed node UI.
```

Comment:
We share URNs to get projects, but we only talk about the id.

Reply:
Isn't the URN the projectID?

Reply:
Nope, the urn contains it, the current format for git pased projects is: `rad:git:<project_id>`.

Reply:
Gotcha. Originally, the project URN was renamed (in the app) to project handle (for simplicity & user friendliness). Then it went up in the air what this *actually* should be called. 

Seeing the format, I agree that projectID *doesn't* make sense, so I believe we should switch back to project handle.

Reply:
I've proposed a resolution to this topic in [#1206](https://github.com/radicle-dev/radicle-upstream/pull/1206) and implemented changes in my latest commits.